token_server/README.md

```diff
@@ -34,9 +34,59 @@ unless it is getting traffic.
 [Dashboard](https://console.cloud.google.com/appengine?folder=&organizationId=838088520005&project=mapping-crisis).
 You must be logged in as `gd-earthengine-user@givedirectly.org` to access it.
 
-## Run server on Amazon EC2
+### Set up new Google Cloud instance
+* Most of the necessary work needs to be done to set up the mapping page to work
+properly in the first place. The only additional work is to create a service
+account that is whitelisted for EarthEngine access (currently
+`earthengine-token-provider`) and give it the "Service Account Token Creator" role
+in the
+[IAM page](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis).
 
-`¯\_(ツ)_/¯`
+* If you are serving tokens from Google App Engine, and not from Amazon, then
+you should also give the default AppEngine service account
+(currently `mapping-crisis@appspot.gserviceaccount.com`) the "Service Account
+Token Creator" role, so that it can create tokens for the service account you
+created above.
+
+## Run server on Amazon Elastic Beanstalk
+
+* Deploy using [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
+Since the project already exists on Amazon Elastic Beanstalk, you just need to
+run `eb deploy` after running `eb init` in order to log in.
+
+* The server URL will be entered here once we have GiveDirectly's account.
+
+* See the [console](https://console.aws.amazon.com/) for more information. We
+are using Elastic Beanstalk.
+
+### Set up new Amazon Elastic Beanstalk instance
```

Comment:
Can you add a when/why you would set up a new instance?

Reply:
Well, Meylakh is about to set up a new instance. It's not routine: the main use case would be if a new user was setting up or they were changing their Amazon account for some reason. I expect that if someone comes to this section, it's because they have to do it, for whatever reason. Added a line.